br/pkg/task/restore.go

```diff
@@ -271,16 +282,16 @@ func CheckNewCollationEnable(
 	backupNewCollationEnable string,
 	g glue.Glue,
 	storage kv.Storage,
-	CheckRequirements bool,
+	SkipCollactionCheck bool,
 ) error {
 	if backupNewCollationEnable == "" {
-		if CheckRequirements {
+		if !SkipCollactionCheck {
 			return errors.Annotatef(berrors.ErrUnknown,
 				"NewCollactionEnable not found in backupmeta. "+
 					"if you ensure the NewCollactionEnable config of backup cluster is as same as restore cluster, "+
```

Comment:
its better give a clear name 'new_collations_enabled_on_first_bootstrap' in error